website/docs/introduction.md

```diff
@@ -40,7 +50,8 @@ Welcome to the JBrowse 2 documentation.
 
```

Comment:
The link to the Jupyter docs on this page is broken now, should link to docs/embedded_components/#jbrowse-jupyter

Reply:
i link to the external site directly now and to jbrowser on the introduction